src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -65,11 +66,12 @@ const OptionRowLHN = (props) => {
         : styles.sidebarLinkText;
     const textUnreadStyle = optionItem.isUnread
         ? [textStyle, styles.sidebarLinkTextUnread] : [textStyle];
-    const displayNameStyle = StyleUtils.combineStyles(props.viewMode === CONST.OPTION_MODE.COMPACT
-        ? [styles.optionDisplayName, ...textUnreadStyle, styles.optionDisplayNameCompact, styles.mr2]
-        : [styles.optionDisplayName, ...textUnreadStyle], props.style);
+    const displayNameStyle = StyleUtils.combineStyles([styles.optionDisplayName, styles.optionDisplayNameCompact, ...textUnreadStyle], props.style);
```

Comment:
Ah why are we merging `styles.optionDisplayName` and `styles.optionDisplayNameCompact` here? Originally we were only appending `styles.optionDisplayNameCompact`  if `props.viewMode === CONST.OPTION_MODE.COMPACT`

Reply:
When the `viewMode` is **Compact**, we put the latest chat text next to `DisplayName` and we append the `style.optionDisplayNameCompact` to make sure DisplayName will take the whole available width. Now, we have `TextPill` next to `DisplayName`, that's why I append it no matter what the `viewMode` is (if we don't do this, the `DisplayName` will get truncated like @shawnborton previously mentioned).

Reply:
Any update on this?

Reply:
Oh thanks for the explanation @bernhardoj, sorry for the delay here